src/main/java/storagesystem/model/StoreIT.java

```diff
@@ -64,16 +67,18 @@ private static void mockData() {
         reservations.add(res);
         reservations.add(res2);
 
-        organisations.add(informationsteknik);
-        organisations.add(data);
-        setCurrentOrganisation(informationsteknik);
-
         tempTeam.addItemToInventory(mockItem);
         tempTeam.addItemToInventory(mockItem2);
     }
 
-    public static void setCurrentUser(User currentUser) {
-        StoreIT.currentUser = currentUser;
+    public static void setCurrentUser(String username) {
```

Comment:
Javadoc? Doesn't really seam necessary, but at the same time we don't have an explanation of currentUser

Reply:
Self-explanatory imo